### DISCUSSION TITLE
[Q&A] 在哪里可以看到自己部署的LobeChat版本？

### DISCUSSION THREAD

**Discussion by WyInnovate** [lobehub/lobehub#852] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

如题，在LobeChat中如何查看当前的版本？

**Comment by ipengyo** [comment, marked as answer]
在【设置】界面底部可以看到当前的部署版本。

![image](https://github.com/lobehub/lobe-chat/assets/9323722/e8ea5d5d-88ad-4b86-872e-655e4c6d3589)

**Reply by WyInnovate** [reply]
感谢，看到了

[a comment by an automated account is not shown]
